filebeat/tests/system/test_shutdown.py

```diff
@@ -43,27 +42,31 @@ def test_shutdown_wait_ok(self):
 
         # Wait until first flush
         self.wait_until(
-            lambda: self.log_contains_count("Flushing spooler") > 1,
+            lambda: self.log_contains_count("Publish event") > 200,
```

Comment:
The problem with setting here 200 instead of one is that the likelyhood of all log lines being already read increases. I want to try to shut down as early as possible. What is the reason you increased this one?